tasks/arrays.yml

```diff
@@ -41,20 +41,27 @@
   - item.state|lower == "present"
   - item.filesystem is defined
 
-# Mounting raid arrays
-- name: arrays | Mounting Array(s)
-  mount:
-    name: "{{ item.mountpoint }}"
-    src: "/dev/{{ item.name }}"
-    fstype: "{{ item.filesystem }}"
-    state: "mounted"
-    opts: "{{ item.opts | default(omit) }}"
+
+# Get UUID of raid device
+- name: arrays | Get UUID of Array(s)
+  ansible.builtin.command: "lsblk /dev/{{ item.name }} -no UUID"
```

Comment:
Given that re-running this role may result in changing the mount, I do wonder how much this differs from just using the name.